### PR TITLE
Removing old miners and adding CCMINER_OPTS usage

### PR DESCRIPTION
**Pull request by delawr0190**
Removing cuda 8 miners and fixing a few things that I found while testing.  Not entirely sure about the eval in 0miner that's wrapping the jq call.  I basically reverted that back to what it was before the $(eval got nuked and things started working again.

**Can somebody please** go through 3main and remove the usage of the miners that I nuked?  I think that all SPccminer usages can safely be replaced with TPccminer.  I didn't touch anything there because I'm unable to test NiceHash or autoswitching (don't use them).

### CODE REVIEW

File: 0miner [hunk 1]
```diff
@@ -190,11 +190,11 @@ then
   pm_manifest=${mpath}/${!xversion}/nvoc-miner.json
   
   # parse device enable params
-  dev_enable_arg="$(jq -r .devlist_argument ${pm_manifest})"
+  dev_enable_arg=$(eval $(jq -r .devlist_argument ${pm_manifest}))
```
Comment (LuKePicci) [inline 0miner:193]: this doesn't work, .devlist_argument does not contain an executable command, therefore you get somewhat equivalent to this `dev_enable_arg=$(eval --cuda_devices)` which does gives token error with eval -- as long as -- is not something evaluable.
As a side effect the if block does not get executed properly.

File: 0miner [hunk 2]
```diff
@@ -260,7 +260,7 @@ then
   OPTS="$(dgh_enabled_devices '--devices ' ',' 'N' "$OPTS")"
   UCCALGO="-a ${ALGO,,}"
   HCD="${NVOC}"/miners/"${!xminer}"/"${!xversion}"/ccminer
-  eval $LAUNCH $HCD $UCCALGO -o ${!xproto}://"${!xpool}":"${!xport}" -u "${!xaddr}""${!xwallet}""${!xwork}" -p "$MINER_PWD" -i "${!xintensity}" $OPTS
+  eval $LAUNCH $HCD $UCCALGO -o ${!xproto}://"${!xpool}":"${!xport}" -u "${!xaddr}""${!xwallet}""${!xwork}" -p "$MINER_PWD" -i "${!xintensity}" $OPTS $CCMINER_OPTS
```
Comment (LuKePicci) [inline 0miner:263]: CCMINER_OPTS has to be deprecated and removed as well. If someone still likes to create a common CCMINER_OPTS variable this should be done in 1bash, then set every TPccminer_OPTS, XXccminer_OPTS to $CCMINER_OPTS if desired.

File: 1bash.template [hunk 3]
```diff
@@ -735,8 +733,8 @@ Z_EWBF_VERSION="recommended"                       # "latest", "recommended" or
 ZENEMYminer_OPTS=""
 ZENEMYminer_VERSION="recommended"                  # "latest", "recommended" or the version folder name.
 
-CCMINER_OPTS="-b 127.0.0.1:4028"                   # ccminer optional arguments. Add "-b 0.0.0.0:4028", or "-b $IPW:42000" if you want 
-                                                   # web info (first one allows remote monitoring by default)
+CCMINER_OPTS="-b 127.0.0.1:4068"                   # ccminer optional arguments. Add "-b 0.0.0.0:4068" if you want
```
Comment (LuKePicci) [inline 1bash.template:736]: See previous comment for 0miner@263

**Review by LuKePicci — CHANGES_REQUESTED** [review]

File: 0miner (hunk 1 above)
Comment (delawr0190) [reply 0miner:193]: I can give it a shot without this in place.  I'm not entirely sure what's going on here, but I know that when I ran my tests, this was changed and miners weren't starting.  This is basically me just putting it back to what it was and then things started running.

**Comment by LuKePicci** [comment]
These changes have been picked and included in #253

**Comment by papampi** [comment]
@delawr0190 
You still need this branch or we can remove it?

**Pull request closed without merging**